CMakeLists.txt

```diff
@@ -9,6 +9,12 @@ endif()
 find_package(Kokkos 4.3 REQUIRED CONFIG)
 message(STATUS "Found Kokkos: ${Kokkos_DIR} (version \"${Kokkos_VERSION}\")")
 
+# We use minimum compiler versions from Kokkos, with the exception for NVCC (>= 11.5)
+SET(ARBORX_NVCC_MINIMUM 11.5.0)
+if(Kokkos_CXX_COMPILER_ID STREQUAL "NVIDIA" AND Kokkos_CXX_COMPILER_VERSION VERSION_LESS ${ARBORX_NVCC_MINIMUM})
```

Comment:
Have you tested that `Kokkos_CXX_COMPILER_ID` is `NVIDIA` when configuring `Kokkos` with `Cuda` support (see https://github.com/kokkos/kokkos/issues/7305 for why this might be different from the `Kokkos` configuration)?

Reply:
It might be better to use `CUDAToolkit_VERSION`.

Reply:
I tested in with `nvcc_wrapper` using two CUDA versions, 11.1 and 11.5. Would that be enough, or do I need to test some other combination?

Reply:
Yeah, it seems to work fine. I also tested on my side.